### ISSUE TITLE
Classification name but not annotation name serialized/deserialized to/from json

### ISSUE THREAD

**Issue by NHPatterson** [qupath/qupath#549]
## Possible bug report

**Describe the bug**

Serialization to geoJSON following protocol described in docs does not add annotation names, but only annotation class. Re-importing this geoJSON, the annotation name is lost. This may be intended behavior or a geoJSON limitation.

**To Reproduce**
create annotation of arbitrary class, and add a custom name like in image below:
![image](https://user-images.githubusercontent.com/17855764/86157253-1f0d7f00-bad5-11ea-9e65-152499d0b98e.png)

Export with this script (not a java/groovy expert here..)
```groovy
import qupath.lib.gui.dialogs.Dialogs;

// Get current image data
def imageData = getCurrentImageData();
def server = imageData.getServer();

// output dir will be prompted
String out_dir = Dialogs.promptForDirectory(null)

// get image name as file path
String fp = server.getURIs();
fp = fp.replace("[file:","")
fp = fp[fp.lastIndexOf('/')+1..-1];
fp = fp.take(fp.lastIndexOf('.'));

fp = out_dir  + '\\' + fp +'_annotations.json';

// grab annotations
def annotations = getAnnotationObjects();
boolean prettyPrint = true;

// prepare json writer
def gson = GsonTools.getInstance(prettyPrint);
String json = gson.toJson(annotations);
FileWriter writer = new FileWriter(fp);

// write to file
writer.write(json);
writer.close();
```
produces a json like this, where proprties only contains "Classification -> Name : test".
```json
[
  {
    "type": "Feature",
    "id": "PathAnnotationObject",
    "geometry": {
      "type": "Polygon",
      "coordinates": [
        [
          [1254, 4069],
          [1239, 4075],
          [1239, 4078],
          [1236, 4078],
          [1006, 4536],
          [1009, 4542],
          [1470, 5144],
          [1473, 5138],
          [1966, 5198],
          [2047, 4770],
          [1892, 4270],
          [1889, 4267],
          [1883, 4261],
          [1880, 4258],
          [1272, 4069],
          [1254, 4069]
        ]
      ]
    },
    "properties": {
      "classification": {
        "name": "test",
        "colorRGB": -8245601
      },
      "isLocked": false,
      "measurements": []
    }
  }
]
```

Re-import annotation using like so:
```groovy
// import libraries
import qupath.lib.io.GsonTools

// instantiate tools
def gson=GsonTools.getInstance(true);
// prepare template
def type = new com.google.gson.reflect.TypeToken<List<qupath.lib.objects.PathObject>>() {}.getType();

def json_fp = promptForFile(null)
// read annotations
bufferedReader = new BufferedReader(new FileReader(json_fp));
deserializedAnnotations = gson.fromJson(bufferedReader.text, type); 

// add to dataset
addObjects(deserializedAnnotations);
```

After removing the test annotation (to see underlying annotation)...
![image](https://user-images.githubusercontent.com/17855764/86157573-9ba05d80-bad5-11ea-8d87-ca215b14eb4b.png)

**Expected behavior**
Annotation name is embedded/readable from geoJSON.

**Screenshots**
See description.

**Desktop (please complete the following information):**
 - OS: Windows10
 - QuPath Version 0.2.0

**Comment by petebankhead** [comment]
Thanks, I think at the time I wrote the JSON serialization names just weren't terribly important. They are a bit more useful now that they can be displayed on the image - I agree they should probably be supported in JSON. Will look into this soon.

**Comment by petebankhead** [comment]
This should now be fixed, and will be included in the upcoming v0.2.2 release.
Please reopen the issue if you find any problem with it.
